### PR TITLE
Add Terp Network: Testnet Resources, Delete Chronic Network Resources

### PR DESCRIPTION
**Pull request by discoverdefiteam**
Chronic Network Project was killed due to corrupted distribution of ownership. 
Also included is testnet registry resources for athena-1, the testnet for [Terp Network.](https://docs.terp.network)

### CODE REVIEW

File: testnets/terpnettestnet/assetlist.json [hunk 1]
```diff
@@ -0,0 +1,50 @@
+{
+    "$schema": "../assetlist.schema.json",
+    "chain_name": "TerpNet",
```
Comment (JeremyParish69) [inline testnets/terpnettestnet/assetlist.json:3]: Please change to 'terpnettestnet' to match folder name exactly.

File: testnets/terpnettestnet/chain.json [hunk 2]
```diff
@@ -0,0 +1,84 @@
+{
+  "chain_name": "TerpNet",
```
Comment (JeremyParish69) [inline testnets/terpnettestnet/chain.json:2]: Please change to 'terpnettestnet' to match folder name exactly.

File: testnets/terpnettestnet/chain.json [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+  "status": "live",
+  "network_type": "testnet",
+  "pretty_name": "Terp-Net",
+  "chain_id": "athena-1",
+  "bech32_prefix": "terp",
+  "daemon_name": "terp",
+  "node_home": "$HOME/.terp",
+  "genesis": {
```
Comment (JeremyParish69) [inline testnets/terpnettestnet/chain.json:10]: please move genesis to the 'codebase' object

**Comment by JeremyParish69** [comment]
For now let's set Chronic Network's network status to 'killed' rather than delete it all. At least until we figure out how to best handle killed chain data (which might go to its own folder).

File: testnets/terpnettestnet/assetlist.json [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            "denom": "persy",
+            "exponent": 6
+          }
+        ],
+        "base": "upersyx",
+        "name": "Persy Testnet Gas Token",
+        "display": "persy",
+        "symbol": "PERSY",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terpnet/images/persy.png",
```
Comment (JeremyParish69) [inline testnets/terpnettestnet/assetlist.json:44]: Can you upload these files, too, please, so they exist?
(i.e., terp.png, terp.svg, persy.png, persy.svg)

File: testnets/terpnettestnet/assetlist.json (hunk 1 above)
Comment (JeremyParish69) [reply testnets/terpnettestnet/assetlist.json:3]: may this was missed. still needs correcting to 'testnettestnet'

File: testnets/terpnettestnet/assetlist.json (hunk 4 above)
Comment (discoverdefiteam) [reply testnets/terpnettestnet/assetlist.json]: got it, thank you.

File: testnets/terpnettestnet/assetlist.json [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+            "exponent": 0
+          },
+          {
+            "denom": "terpx",
+            "exponent": 6
+          }
+        ],
+        "base": "uterpx",
+        "name": "Terp Testnet Token",
+        "display": "terp",
```
Comment (JeremyParish69) [inline testnets/terpnettestnet/assetlist.json:19]: this needs to match one of the denom units. Should be "terpx"

File: testnets/terpnettestnet/assetlist.json [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+            "exponent": 0
+          },
+          {
+            "denom": "persyx",
+            "exponent": 6
+          }
+        ],
+        "base": "upersyx",
+        "name": "Persy Testnet Gas Token",
+        "display": "persy",
```
Comment (JeremyParish69) [inline testnets/terpnettestnet/assetlist.json:39]: should be "persyx"

File: testnets/terpnettestnet/assetlist.json [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+            "denom": "terpx",
+            "exponent": 6
+          }
+        ],
+        "base": "uterpx",
+        "name": "Terp Testnet Token",
+        "display": "terp",
+        "symbol": "TERP",
+        "logo_URIs": {
+          "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/terpnet/images/terp.png"
```
Comment (JeremyParish69) [inline testnets/terpnettestnet/assetlist.json:22]: this location does not exist. For now, should be within the /testnets/ folder

File: testnets/terpnettestnet/chain.json [hunk 8]
```diff
@@ -0,0 +1,83 @@
...
+  "peers": {
+    "seeds": [
+      
+      {
+        "id": "9b9ff10893b11aa5588170b289222d4bb6785d97",
+        "address": "23.81.114.228:26656"
+      }
+    ],
+    "persistent_peers": [
+
```
Comment (JeremyParish69) [inline testnets/terpnettestnet/chain.json:35]: got a few of these random extra blank lines that doesn't really need to be there... not a big deal but it'd be cleanest to not be there

File: testnets/terpnettestnet/assetlist.json [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+          {
+            "denom": "persyx",
+            "exponent": 6
+          }
+        ],
+        "base": "upersyx",
+        "name": "Persy Testnet Gas Token",
+        "display": "persy",
+        "symbol": "PERSY",
+        "logo_URIs": {}
```
Comment (JeremyParish69) [inline testnets/terpnettestnet/assetlist.json:41]: no logo?

Comment (discoverdefiteam) [reply testnets/terpnettestnet/assetlist.json:41]: not as of yet, will likely have one by the time we are ready to update the chain registry with main net resources.

**Pull request merged**